### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by mahbubul14**
## Project requirements:

* Remove all hardcoded items from the tasks array.
* Create a new JavaScript file for the new functionality.
* Implement a function for adding a new task (add a new element to the array).
* Implement a function for editing task descriptions.
* Implement a function for deleting a task (remove an element from the array).
* Implement a function for the "Clear all completed" button (use filter() method).
* By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal 5).
* Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique.
* All changes to the To-Do List should be saved in local storage.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 1]
```diff
@@ -52,4 +68,17 @@ function checkStorage() {
   }
   return taskStorage;
 }
+
+addBtn.addEventListener('click', (e) => {
+  e.preventDefault();
+  add(checkStorage(), document.getElementById('task').value);
+  form.reset();
```
Comment (mikenath223) [inline src/index.js:71]: - [x] The functionality to remove a todo item doesn't work as clicking on the delete icon has no effect, please fix it.

File: src/index.js [hunk 2]
```diff
@@ -1,47 +1,63 @@
...
-    const threeDot = document.createElement('i');
-    threeDot.className = 'fas fa-ellipsis-v';
+    // const threeDot = document.createElement('i');
+    const removeIcon = document.createElement('i');
+
+    checkbox.setAttribute('type', 'checkbox');
+    checkbox.id = 'checkbox';
+    checkbox.checked = isDone(task);
+    // threeDot.className = 'fas fa-ellipsis-v';
+    removeIcon.className = 'far fa-trash-alt';
```
Comment (mikenath223) [inline src/index.js]: - [x] Kindly remove this and all other commented code, as commented code tends to make the project seem untidy.

File: src/index.js [hunk 3]
```diff
@@ -1,47 +1,63 @@
...
+
+    itemText.addEventListener('click', () => {
+      itemText.setAttribute('contenteditable', 'true');
+    });
+
+    itemText.addEventListener('keypress', (e) => {
+      if (e.key === 'Enter') {
+        edit(task, itemText.innerText);
+        window.location.reload();
+      }
```
Comment (mikenath223) [inline src/index.js:54]: - [ ] Kindly change all regular functions to es6 arrow function syntax following the JavaScript best practices guide.

File: src/index.js (hunk 1 above)
Comment (mahbubul14) [reply src/index.js:71]: - [ ] the mentioned code shows add button but you asked to fix the remove button. I am a little bit confused. @mikenath223

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**